mistletoe/base_renderer.py

```diff
@@ -41,9 +41,11 @@ class BaseRenderer(object):
                         parsing process.
     """
     _parse_name = re.compile(r"([A-Z][a-z]+|[A-Z]+(?![a-z]))")
+    auto_header_ids = False
```

Comment:
Note to self: Place within `HTMLAttributesRenderer` class as a configurable property